docs/usage.md

```diff
@@ -516,6 +546,11 @@ Atheris example:
     $ cp casr/tests/casr_tests/python/yaml_fuzzer.py .
     $ casr-libfuzzer -i casr/tests/casr_tests/casrep/atheris_crashes_ruamel_yaml -o casr/tests/tmp_tests_casr/casr_libfuzzer_atheris_out -- ./yaml_fuzzer.py
 
+Jazzer.js example:
+    $ unzip casr/tests/casr_tests/js/crashes.zip -d crashes
+    $ cp casr/tests/casr_tests/js/test_casr_libfuzzer_jazzer_js.js .
```

Comment:
do we need cp? Or could just use this path in the next command?

Reply:
Done, fixed also for Atheris